examples/nodejs/eventbridge/infrastructure/lib/eventbridge-stack.ts

```diff
@@ -0,0 +1,220 @@
+      type: 'String',
+      description: 'The API key for Momento.',
+    });
+
+    const gameScoreDemoTable = new dynamodb.Table(this, "game-scores-demo-table", {
+      tableName: "game-scores-demo",
+      partitionKey: { name: "GameId", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "GamerTag", type: dynamodb.AttributeType.STRING },
+      stream: dynamodb.StreamViewType.NEW_AND_OLD_IMAGES,
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+    });
+
+    const logGroup = new logs.LogGroup(this, "AccessLogs", {
+      retention: 90,
+      logGroupName: cdk.Fn.sub(
+        `game-scores-demo-logs-\${AWS::Region}`,
+      ),
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+    });
+
+    const apiKeySecret = new Secret(this, 'MomentoApiKey', {
+      secretName: 'momento-eventbridge-api-key',
+      secretStringValue: new cdk.SecretValue(momentoApiKeyParameter.valueAsString),
+    });
+    const connection = new events.Connection(this, 'game-scores-demo-connection', {
+      connectionName: 'game-scores-demo-connection',
+      authorization: events.Authorization.apiKey('Authorization', cdk.SecretValue.secretsManager(apiKeySecret.secretName)),
+      description: 'Connection with API Key Authorization',
+    });
+
+    const cachePutApiDestination = new events.ApiDestination(this, "game-scores-demo-cache-put-api-destination", {
+      apiDestinationName: "game-scores-demo-cache-put-api-destination",
+      connection,
+      endpoint: "https://api.cache.cell-alpha-dev.preprod.a.momentohq.com/cache/*",
+      description: "Alpha Cache Set API",
+      httpMethod: events.HttpMethod.PUT,
+    });
+
+    const topicPublishApiDestination = new events.ApiDestination(this, "game-scores-demo-topic-publish-api-destination", {
+      apiDestinationName: "game-scores-demo-topic-publish-api-destination",
+      connection,
+      endpoint: "https://api.cache.cell-alpha-dev.preprod.a.momentohq.com/topics/*/*",
+      description: "Alpha Topic Publish API",
+      httpMethod: events.HttpMethod.POST,
+    });
+
+    const cacheDeleteApiDestination = new events.ApiDestination(this, "game-scores-demo-cache-delete-api-destination", {
+      apiDestinationName: "game-scores-demo-cache-delete-api-destination",
+      connection,
+      endpoint: "https://api.cache.cell-alpha-dev.preprod.a.momentohq.com/cache/*",
+      description: "Alpha Cache Delete API",
+      httpMethod: events.HttpMethod.DELETE,
+    });
+
+    const role = new iam.Role(this, "AmazonEventBridgePipeGameDemoEventToMomentoCache", {
+      roleName: "AmazonEventBridgePipeGameDemoEventToMomentoCache",
+      assumedBy: new iam.ServicePrincipal("pipes.amazonaws.com"),
+    });
+
+    const deadLetterQueue = new sqs.Queue(this, "DeadLetterQueue", {
+      queueName: "game-scores-demo-dlq",
+      retentionPeriod: cdk.Duration.days(14),
+    });
+
+    const cachePutCfnPipe = new pipes.CfnPipe(this, "game-scores-demo-cache-put-pipe", {
```

Comment:
just curious here - usually when I see these ones prefixed with `Cfn*` it means that you are using the raw cloudformation resources directly rather than the higher-level abstractions that CDK provides. Sometimes it's because they don't have a higher level abstraction, sometimes it's because the high level abstraction doesn't have all of the features you need. What's the case here?

Reply:
CDK doesn't provide constructs to create eventbridge resource. Therefore, had to use raw cloudformation resource: `CfnPipes` directly. 
[Documentation for Ref ](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_pipes.CfnPipe.html)

Reply:
cool, makes sense.